examples/babyai/config/tensorzero.toml

```diff
@@ -9,73 +9,57 @@ routing = ["openai"]
-type = "chat_completion"
-weight = 1.0
-model = "my_gpt_4o_mini"
-system_template = "functions/generate_action/baseline/system.minijinja"
-user_template = "functions/generate_action/baseline/user.minijinja"
+[functions.act]
+type = "json"
+system_schema = "functions/act/system_schema.json"
+user_schema = "functions/act/user_schema.json"
+output_schema = "functions/act/output_schema.json"
 
-[functions.generate_action.variants.no_mission]
+[functions.act.variants.baseline]
 type = "chat_completion"
 weight = 1.0
 model = "my_gpt_4o_mini"
-system_template = "functions/generate_action/no_mission/system.minijinja"
-user_template = "functions/generate_action/no_mission/user.minijinja"
+system_template = "functions/act/baseline/system.minijinja"
+user_template = "functions/act/baseline/user.minijinja"
 
-[functions.generate_action.variants.history]
+[functions.act.variants.history]
 type = "chat_completion"
 weight = 1.0
 model = "my_gpt_4o_mini"
-system_template = "functions/generate_action/history/system.minijinja"
-user_template = "functions/generate_action/history/user.minijinja"
+system_template = "functions/act/history/system.minijinja"
+user_template = "functions/act/history/user.minijinja"
 
-[functions.generate_action.variants.reasoning]
+[functions.act.variants.reasoning]
 type = "chat_completion"
 weight = 1.0
 model = "my_gpt_4o_mini"
-system_template = "functions/generate_action/reasoning/system.minijinja"
-user_template = "functions/generate_action/reasoning/user.minijinja"
+system_template = "functions/act/reasoning/system.minijinja"
+user_template = "functions/act/reasoning/user.minijinja"
 
-[functions.generate_action.variants.history_and_reasoning]
+[functions.act.variants.history_and_reasoning]
 type = "chat_completion"
 weight = 1.0
 model = "my_gpt_4o_mini"
-system_template = "functions/generate_action/history_and_reasoning/system.minijinja"
-user_template = "functions/generate_action/history_and_reasoning/user.minijinja"
+system_template = "functions/act/history_and_reasoning/system.minijinja"
+user_template = "functions/act/history_and_reasoning/user.minijinja"
 
-[functions.generate_action.variants.history_ft]
-type = "chat_completion"
-weight = 0
-model = "ft:gpt-4o-mini-2024-07-18:tensorzero::ApNUdVRd"
-system_template = "functions/generate_action/history/system.minijinja"
-user_template = "functions/generate_action/history/user.minijinja"
-
-[functions.extract_action]
-type = "json"
-output_schema = "functions/extract_action/output_schema.json"
-
-[functions.extract_action.variants.simple_variant]
+[functions.act.variants.history_and_reasoning_ft]
 type = "chat_completion"
 weight = 1.0
-model = "my_gpt_4o_mini"
-system_template = "functions/extract_action/simple_variant/system.minijinja"
+model = "ft:gpt-4o-mini-2024-07-18:tensorzero::Arsyj2TT"
+system_template = "functions/act/history_and_reasoning/system.minijinja"
+user_template = "functions/act/history_and_reasoning/user.minijinja"
 
```

Comment:
we should remove this as well, people can add their own variants for fine-tuning